src/CoreBluetooth/Enums.cs

```diff
@@ -23,6 +24,17 @@ public enum CBManagerState : nint {
 		PoweredOff,
 		PoweredOn
 	}
+#else
```

Comment:
are the values are needed at build time ?

Might be better to have if `#if` only on the `enum` line - that way there's not chance of only one side of the (identical) values being updated in the future